CHANGELOG.txt

```diff
@@ -24,6 +24,7 @@ Changelog
  * `StreamField` field blocks now accept a `validators` argument (Tom Usher)
  * Added edit / delete buttons to snippet index and "don't delete" option to confirmation screen, for consistency with pages (Kevin Howbrook)
  * Added support for Markdown shortcuts for inline formatting in rich text editor, e.g. `**` for bold, `_` for italic, etc. (Thibaud Colas)
+ * Updated the templates documentation to include `ajax_template` (Kalob Taulien)
```

Comment:
We don't usually include documentation updates in the changelog, as there are so many of them.

(Also, as a heads-up - changelog notes should also go on the release notes page in `/docs/releases` - usually it's more convenient to add these at the point of merging the PR, to avoid merge conflicts.)

Reply:
@gasman that's good to know, thanks! And any future PR's I'll make the `/docs/releases` gets updated.